### PR TITLE
multiple full repairs with lcs

### PR DESCRIPTION
**Pull request by krummas**
CASSANDRA-11172

### CODE REVIEW

File: repair_tests/incremental_repair_test.py [hunk 1]
```diff
@@ -279,6 +279,19 @@ def compaction_test(self):
         for x in range(0, 150):
             assert_one(session, "select val from tab where key =" + str(x), [1])
 
+    def multiple_full_repairs_lcs_test(self):
+        """
+        @jira_ticket CASSANDRA-11172
+        """
+        cluster = self.cluster
+        cluster.populate(2).start(wait_for_binary_proto=True)
+        [node1, node2] = cluster.nodelist()
```
Comment (ptnapoleon) [inline repair_tests/incremental_repair_test.py]: The `[]` here are unnecessary. You can just `node1, node2 = cluster.nodelist()`

**Comment by ptnapoleon** [comment]
+1 after style nit

**Comment by mambocab** [comment]
Could you also add some information to the test's docstring explaining the test's relationship to `CASSANDRA-11172` (does it test a new feature? a regression?) and a brief description of the test's operation and the desired C* behavior?

**Comment by krummas** [comment]
ok, nit/comment fixed and merged

**Pull request merged**